pkg/utils/retry.go

```diff
@@ -39,7 +43,22 @@ func NewRetryRunner(maxTimes int, delay time.Duration, expBackoff bool) *retryRu
 	}
 }
 
+// NewProbabilisticRetryRunner returns a new runner that sets delay to time.Duration(rand.Int63n(f.interval)+1) * time.Millisecond
+func NewProbabilisticRetryRunner(maxTimes int, interval int64, expBackoff bool) *retryRunner {
```

Comment:
We could use `time.Duration` as `interval`?

Reply:
interval is used to compute delay as described in the comments. So, it is a different beast, I think. No?

Reply:
No instead of passing the milliseconds as parameter (lets say `5`), you can pass `5 * time.Millisecond` that is more readable

Reply:
yeah, but in which range do I sample?